index.html

```diff
@@ -5,6 +5,7 @@
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Star Wars Info Hub</title>
+    <link rel="stylesheet" href="./src/styles.css" />
```

Comment:
This is definitely how we'd do something in a pure HTML setting, but Vite's react template is pre-configured with the ability to import css files as javascript. This allows you to "scope" css files to specific components instead of having to use global styles for everything. Either way, for global styles like font and other things, I'd still recommend importing this instead from the `main.tsx` or `App.tsx` files.

Reply:
this still should get removed